### ISSUE TITLE
Fix Che dashboard build

### ISSUE THREAD

**Issue by garagatyi** [redhat-developer/rh-che#958]
**Issue problem:** 
UD is no longer builds in RH-CHE (against latest snapshot) after the changes in latest upstream.
The error in build logs:
<details>

```
10:37:31      [exec] [91mnpm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
10:37:31      [exec] npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
10:37:31      [exec] [0m[91mnpm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.3 (node_modules/sane/node_modules/fsevents):
10:37:31      [exec] [0m[91mnpm WARN notsup[0m[91m SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
10:37:31      [exec] [0m[91mnpm WARN angular-gravatar@0.2.4 requires a peer of grunt-cli@0.1.13 but none was installed.
10:37:31      [exec] [0m[91mnpm WARN[0m[91m file-loader@1.1.11 requires a peer of webpack@^2.0.0 || ^3.0.0 || ^4.0.0 but none was installed.
10:37:31      [exec] npm WARN ts-jest@22.4.6 requires a peer of typescript@2.x but none was installed.
10:37:31      [exec] [0m[91mnpm WARN che-dashboard@4.0.0 No description
10:37:31      [exec] [0m[91mnpm WARN optional SKIPPING OPTIONAL DEPENDENCY: bufferutil@1.1.0 (node_modules/bufferutil):
10:37:31      [exec] npm WARN optional[0m[91m SKIPPING OPTIONAL DEPENDENCY: bufferutil@1.1.0 install: `node-gyp rebuild`
10:37:31      [exec] npm WARN optional[0m[91m SKIPPING OPTIONAL DEPENDENCY: Exit status 1
10:37:31      [exec] [0m[91mnpm WARN optional SKIPPING OPTIONAL DEPENDENCY: utf-8-validate@1.1.0 (node_modules/utf-8-validate):
10:37:31      [exec] npm WARN[0m[91m optional SKIPPING OPTIONAL DEPENDENCY: utf-8-validate@1.1.0 install: `node-gyp rebuild`
10:37:31      [exec] npm [0m[91mWARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
10:37:31      [exec] [0m[91mnpm info ok 
10:38:05      [exec] [0mRemoving intermediate container b7a543f6f86b
10:38:05      [exec]  ---> e00c0d4d021f
10:38:05      [exec] Step 6/10 : COPY ${SOURCES_DIR}/bower.json /dashboard/
10:38:05      [exec] COPY failed: stat /var/lib/docker/tmp/docker-builder279638829/target/sources/bower.json: no such file or 
```
</details>
Broken [here](https://github.com/eclipse/che/pull/11301/files): bower is removed, dockerfile is rewritten.

**Red Hat Che version:** 

version: (help/about menu)

 - [ ] I can reproduce it on latest official image

**Reproduction Steps:**

Describe how to reproduce the problem

**Runtime:**

runtime used:
   - [ ] minishift (include output of `minishift version`)
   - [ ] OpenShift.io
   - [ ] Openshift Container Platform (include output of `oc version`)
   - [ ] k8s (include output of `kubectl version`)

**Comment by amisevsk** [comment]
Would probably require just updating the `assembly/fabric8-ide-dashboard-war/Dockerfile`, as it copies bower.json explicitly.

**Comment by amisevsk** [comment]
I've created branch [6.13-SNAPSHOT](https://github.com/redhat-developer/rh-che/tree/6.13-SNAPSHOT) to track necessary changes for building with upstream snapshot builds.

**Comment by amisevsk** [comment]
Fixed in PR #965 

Closing as this issue is subsumed by https://github.com/redhat-developer/rh-che/issues/1001
